src/usr/share/netcheck/netcheck.py

```diff
@@ -776,3 +786,20 @@ def _calculate_next_log_time(self, loop_time):
         Returns the datetime that the logging should occur.
         """
         return loop_time + datetime.timedelta(seconds=self.config['periodic_status_delay'])
+
+    def _update_default_gateway_state(self):
```

Comment:
I'm not sure this name captures the real purpose of the method. I'd probably call this, _check_for_gateway_change.